src/calculateRentalCost.js

```diff
@@ -29,6 +29,18 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let sum;
+  const price = 40;
+
+  if (days >= 7) {
+    sum = days * price - 50;
+  } else if (days >= 3) {
+    sum = days * price - 20;
+  } else if (days < 3) {
+    sum = days * price;
+  }
+
+  return sum;
```

Comment:
```suggestion
  if (days >= 7) {
    return  totalPrice - 50;
  } 
   
  if (days >= 3) {
    return totalPrice - 20;
  }

  return totalPrice;
```

Reply:
not fixed comment about avoiding else/if else .
please refactor it.